### DISCUSSION TITLE
No way to pay

### DISCUSSION THREAD

**Discussion by 21888** [community/community#30521] [Copilot Conversations]
When I bought the Copilot
Web page prompt
Your account is currently locked from purchases. Please update your payment information.

**Comment by Akash1134** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted :)

**Reply by 21888** [reply]
I'm sorry, I'm Chinese, the payment here is different from yours, and I don't quite understand what you are talking about, so I want to know why I can't use Visa to pay, or how to solve the problem, I just want to buy Copilot

**Reply by byp520** [reply]
兄弟没有用paypal吗

**Reply by 21888** [reply]
七十多一个月 这幌子用着咋这么心疼呢

**Reply by 618457** [reply]
更新付款信息没用，去Support Page开个ticket，等客服给你看看statement是不是不正常。（比如有欠款）
